Makefile

```diff
@@ -37,7 +37,7 @@ OBJECTDRAW_REAL = $(filter-out %tobjectdraw.grace, $(OBJECTDRAW))
 PRELUDESOURCEFILES = collectionsPrelude.grace StandardPrelude.grace
 REALSOURCEFILES = $(sort compiler.grace errormessages.grace util.grace ast.grace identifierKinds.grace lexer.grace parser.grace genjs.grace genc.grace stringMap.grace xmodule.grace identifierresolution.grace)
 SOURCEFILES = $(MGSOURCEFILES) $(PRELUDESOURCEFILES)
-STABLE=5585c0e69862e890e55fad7079c674a0c95c200b
+STABLE=1e2f5d98abb4529b811183acfa9f4c89618406e0
```

Comment:
This is rolling back to an older STABLE compiler.  This is almost certainly not what you intended.